aws/amazon-linux-2/docker.pkr.hcl

```diff
@@ -55,6 +55,8 @@ locals {
         "fuse",
         # Install git so we can fetch the source code to be tested, obviously!
         "git",
+        # (Optional) zip is required if any tests create undeclared test outputs
```

Comment:
nit: link to https://bazel.build/reference/test-encyclopedia for more information for folks who don't know what undeclared test outputs are

Reply:
:+1: